src/meta/MetaServiceUtils.cpp

```diff
@@ -20,11 +20,20 @@ const std::string kIndexTable  = "__index__";   // NOLINT
 const std::string kUsersTable  = "__users__";    // NOLINT
 const std::string kRolesTable  = "__roles__";    // NOLINT
 const std::string kConfigsTable = "__configs__"; // NOLINT
+const std::string kLastUpdateTimeTable = "__last_update_time__"; // NOLINT
+const std::string kLeaderDistTable = "__leader_dist__"; // NOLINT
```

Comment:
Can't we save the leader information in parts table or hosts table?

Reply:
OK, the leader information may be more appropriate to put into the hosts table.

Reply:
done!